tests/security/requirement_ports.txt

```diff
@@ -20,6 +20,9 @@
 21501
 22000
 22001
+22002
```

Comment:
It would readable if you add a readme about port and the services who use them.

Reply:
Hi @seagate-sarang-sawant 

Thanks for this suggestion, I will update the text file and corresponding parsing to accommodate comments.
However I will add this suggestions in subsequent PRs.

For this PR, I am not making any further changes, unless something breaks.
I suggest let this test run with nightly tests a couple of times before making any further changes.

Let me know otherwise

Regards
Swanand